src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -93,51 +139,38 @@ public void testInsertAndGet() {
 
   @Test
   public void testUpdateCurrentVersionUuid() {
-    final NamespaceRow namespaceRow = newNamespaceRow();
-    final DatasourceRow datasourceRow = newDatasourceRow();
-    namespaceDao.insert(namespaceRow);
-    datasourceDao.insert(datasourceRow);
-
     final DatasetRow newDatasetRow =
         newDatasetRowWith(namespaceRow.getUuid(), datasourceRow.getUuid());
     final DatasetRow datasetRow = datasetDao.insertAndGet(newDatasetRow).orElse(null);
     assertThat(datasetRow.getCurrentVersionUuid()).isNull();
+    final Instant previousUpdatedAt = datasetRow.getUpdatedAt();
 
     final UUID currentVersionUuid = UUID.randomUUID();
     datasetDao.updateCurrentVersionUuid(datasetRow.getUuid(), currentVersionUuid);
 
     final DatasetRow datasetRowWithVersion = datasetDao.findBy(datasetRow.getUuid()).orElse(null);
     assertThat(datasetRowWithVersion.getCurrentVersionUuid()).isNotNull();
     assertThat(datasetRowWithVersion.getCurrentVersionUuid()).isEqualTo(currentVersionUuid);
+    assertThat(datasetRowWithVersion.getUpdatedAt()).isAfter(previousUpdatedAt);
```

Comment:
Not sure the variable adds much, would be simpler to assert:

```java
assertThat(datasetRowWithVersion.getUpdatedAt()).isAfter(datasetRow.getUpdatedAt());
```

Reply:
true - fixed.